sdcm/cluster.py

```diff
@@ -1367,7 +1367,9 @@ def wait_db_up(self, verbose=True, timeout=3600):
     def is_manager_agent_up(self, port=None):
         port = port if port else self.MANAGER_AGENT_PORT
         # When the agent is IP, it should answer an https request of https://NODE_IP:10001/ping with status code 204
-        response = requests.get(f"https://{normalize_ipv6_url(self.cql_address)}:{port}/ping", verify=False)
+        # Scylla Manager Agent uses Scylla listen/broadcast address - https://manager.docs.scylladb.com/stable/config/
+        # scylla-manager-agent-config.html#:~:text=value.%0A%23auth_token%3A-,%23%20Bind%20REST%20API%20to,-the%20specified%20TCP
+        response = requests.get(f"https://{normalize_ipv6_url(self.ip_address)}:{port}/ping", verify=False)
```

Comment:
if listen/broadcast address is used, why not to take `self.scylla_listen_address`?
So in case test config uses dns, we would test it here too.

Reply:
fixed